enterprise/internal/campaigns/syncer.go

```diff
@@ -307,80 +367,119 @@ func (s *ChangesetSyncer) listAllNonDeletedChangesets(ctx context.Context) (all
 	return all, err
 }
 
-type changesetQueue struct {
-	scheduled chan int64
-	priority  chan int64
+type syncSchedule struct {
```

Comment:
I find this name much easier to reason about, dunno why :)
```suggestion
type scheduledSync struct {
```